src/Data/Fin/Base.agda

```diff
@@ -122,6 +122,14 @@ inject≤ : ∀ {m n} → Fin m → m ℕ.≤ n → Fin n
 inject≤ zero    (s≤s le) = zero
 inject≤ (suc i) (s≤s le) = suc (inject≤ i le)
 
+-- lower₁ "n" _ = "n".
```

Comment:
Should this be `lower₁ "i" _ = "i"` instead?

Reply:
I am actually puzzled by the comment in this module. 

For example, shouldn't the following comment
```agda
-- raise m "n" = "m + n".
raise : ∀ {m} n → Fin m → Fin (n ℕ.+ m)
raise zero    i = i
raise (suc n) i = suc (raise n i)
```
be 
```
-- raise m "n" = "m + n".

Reply:
Yup! Good spot. It'd be great if you could change that as well.

Reply:
Great. I will go through this module later.

Reply:
They should be renamed accordingly now.